OpenSim/ExampleComponents/osimExampleComponentsDLL.h

```diff
@@ -0,0 +1,49 @@
+#ifndef _osimExamplecomponentsDLL_h_
+#define _osimExamplecomponentsDLL_h_
```

Comment:
Ok, I agree: I don't think a file named "osimExamplecomponentsDLL.h" will ever coexist with this one, so this header guard is fine.